src/api/app/services/workflows/filter.rb

```diff
@@ -2,12 +2,12 @@ module Workflows
   class Filter
     def initialize(filters:)
       filters ||= {}
-      @repository_filters = filters[:repositories]
-      @architecture_filters = filters[:architectures]
+      @repository_filters = filters[:repositories].deep_symbolize_keys
```

Comment:
Doing `deep_symbolize_keys` once directly on `filters` would be better.

Reply:
Done